way-cooler/src/callbacks.rs

```diff
@@ -117,8 +117,21 @@ pub extern fn view_request_move(view: WlcView, _dest: &Point) {
     }
 }
 
-pub extern fn view_request_resize(_view: WlcView, _edge: ResizeEdge, _location: &Point) {
-
+pub extern fn view_request_resize(view: WlcView, edge: ResizeEdge, point: &Point) {
```

Comment:
It looks like this PR is behind resizing and the `git mv`s may have screwed it up (might also be annoying on future PRs). Not sure what to do here.